### PR TITLE
Add systemd-collector as default for node exporter

### PR DESCRIPTION
**Pull request by MalloZup**
@witekest  as discussed.

 note I **didn't test this**

### CODE REVIEW

File: prometheus-exporters-formula/prometheus-exporters-formula.changes [hunk 1]
```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Oct  8 13:23:59 UTC 2020 - Dario Maiocchi <dmaiocchi@suse.com>
+
+- Version 0.7.5:
```
Comment (witekest) [inline prometheus-exporters-formula/prometheus-exporters-formula.changes:4]: Could you please bump the version number to 0.8.0? I'll package that together with https://github.com/SUSE/salt-formulas/pull/82

File: prometheus-exporters-formula/prometheus-exporters-formula.spec [hunk 2]
```diff
@@ -18,7 +18,7 @@
 %define fname prometheus-exporters
 %define fdir  %{_datadir}/susemanager/formulas
 Name:           prometheus-exporters-formula
-Version:        0.7.4
+Version:        0.7.5
```
Comment (witekest) [inline prometheus-exporters-formula/prometheus-exporters-formula.spec:21]: 0.8.0 please

**Review by witekest — CHANGES_REQUESTED** [review]
Tested and works as expected enabling _systemd_ metrics in fresh installation of _node-exporter_.

File: prometheus-exporters-formula/prometheus-exporters-formula.changes (hunk 1 above)
Comment (MalloZup) [reply prometheus-exporters-formula/prometheus-exporters-formula.changes:4]: ok

**Comment by witekest** [comment]
Please also update the [Makefile](https://github.com/SUSE/salt-formulas/blob/master/Makefile#L5).

**Review by witekest — APPROVED** [review]

**Pull request merged**